### PR TITLE
CLN: Update .format(...) strings to f-expressions

### PR DESCRIPTION
**Pull request by srinivasreddy**
- [x] xref  #29547
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by alimcmaster1** [comment]
Thanks - please link the issue https://github.com/pandas-dev/pandas/issues/29547

**Comment by alimcmaster1** [comment]
CI failures related to black

```
All done! 💥 💔 💥
2 files would be reformatted, 865 files would be left unchanged.
Checking black formatting DONE
```

Please run `black pandas`

File: asv_bench/benchmarks/gil.py [hunk 1]
```diff
@@ -250,13 +250,11 @@ def setup(self, dtype):
                 np.random.randn(rows, cols), index=date_range("1/1/2000", periods=rows)
             ),
             "object": DataFrame(
-                "foo",
-                index=range(rows),
-                columns=["object%03d".format(i) for i in range(5)],
+                "foo", index=range(rows), columns=[f"object%03d" for i in range(5)],
```
Comment (datapythonista) [inline asv_bench/benchmarks/gil.py]: `i` was ignored, no need for an f-string here I think

File: pandas/core/algorithms.py [hunk 2]
```diff
@@ -1159,7 +1159,7 @@ def compute(self, method):
         n = min(n, narr)
 
         kth_val = algos.kth_smallest(arr.copy(), n - 1)
-        ns, = np.nonzero(arr <= kth_val)
+        (ns,) = np.nonzero(arr <= kth_val)
```
Comment (datapythonista) [inline pandas/core/algorithms.py]: I guess this is from black. Can you use the version in `environment.yml` (`conda install black<=19.3b0`), so no unrelated changes are applied.

**Review by datapythonista — COMMENTED** [review]
Thanks for working on this @srinivasreddy. Please run `black asv_bench/benchmarks/gil.py` and push the changes, so the CI is green and we can merge.

Also, avoid unrelated changes if they are not coming from black.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks for working on this @srinivasreddy

**Comment by jreback** [comment]
thanks @srinivasreddy
